README.md

```diff
@@ -1 +1,39 @@
-# Server
\ No newline at end of file
+# Server
+
+## Access
+You can get access to heat backend by url: [https://heat-backend.herokuapp.com](https://heat-backend.herokuapp.com/).
+
+## Methods
+- */registration*
```

Comment:
Plesae, specify the method type (GET/POST/etc.).

Reply:
Done.